bentoml/_internal/log.py

```diff
@@ -82,6 +82,7 @@ def filter(self, record: logging.LogRecord) -> bool:
 
 def configure_logging():
     # TODO: convert to simple 'logging.basicConfig' after we no longer need the filter
+    logging.captureWarnings(True)
```

Comment:
we use warnings.warn for experimental, instead of logger.warning, hence we want to capture warnings here.

Reply:
make sure we are aware of that this will also affect other libs